ext/standard/tests/array/array_flip.phpt

```diff
@@ -12,26 +12,39 @@ $trans = array("a" => 1,
                0 => "G",
                1 => "h",
                2 => "i");
-$trans = array_flip($trans);
+
+try {
+    $trans = array_flip($trans);
+} catch (\TypeError $e) {
+    echo $e->getMessage() . "\n";
+}
 var_dump($trans);
```

Comment:
this `var_dump` can be removed - as well as the EXPECT output. When an exception occurs the array is not modified - this is known behavior and doesn't need testing as part of this change